src/cli/tc.rs

```diff
@@ -31,9 +32,82 @@ pub async fn run() -> Result<()> {
         Err(e) => Err(e),
       }
     }
+    Command::Init => init().await,
   }
 }
 
+pub async fn init() -> Result<()> {
+  let tailcallrc = r"directive @server(
```

Comment:
We should read the file that's inside of assets folder and use that until #374 is completed.